src/NotificationServices/Softeq.NetKit.Services.SmsNotifications/Abstract/BaseSmsNotification.cs

```diff
@@ -0,0 +1,11 @@
+﻿// Developed by Softeq Development Corporation
+// http://www.softeq.com
+
+namespace Softeq.NetKit.Services.SmsNotifications.Abstract
+{
+    public class BaseSmsNotification : ISmsNotification
+    {
+        public string Recipient { get; set; }
```

Comment:
Should add more descriptive property. Like RecipientPhoneNumber

Reply:
done